.buildkite/pull-requests.json

```diff
@@ -1,27 +1,27 @@
-{
-    "jobs": [{
-        "enabled": true,
-        "pipeline_slug": "fleet-server",
-        "allow_org_users": true,
-        "allowed_repo_permissions": ["admin", "write"],
-        "set_commit_status": true,
-        "build_on_commit": true,
-        "build_on_comment": true,
-        "trigger_comment_regex": "^(?:(?:buildkite\\W+)?(?:build|test)\\W+(?:this|it))|^/test$",
-        "always_trigger_comment_regex": "^(?:(?:buildkite\\W+)?(?:build|test)\\W+(?:this|it))|^/test$",
-        "fail_on_not_mergeable": true
-      },
-      {
-        "enabled": true,
-        "pipeline_slug": "fleet-server-package-mbp",
-        "allow_org_users": true,
-        "allowed_repo_permissions": ["admin", "write"],
-        "set_commit_status": false,
-        "build_on_commit": false,
-        "build_on_comment": true,
-        "trigger_comment_regex": "^(?:(?:buildkite\\W+)?(?:package)\\W+(?:this|it))|^/package$",
-        "always_trigger_comment_regex": "^(?:(?:buildkite\\W+)?(?:package)\\W+(?:this|it))|^/package$",
-
-        "fail_on_not_mergeable": true
-      }]
-}
+{
```

Comment:
what is the change in this file ?

Reply:
In this file was changed just EOL from CRLF to LF